role mapping & small fixes (#39605)
airbytehq/airbyte: docs/enterprise-setup/scaling-airbyte.md

  jobs:
    resources:
      limits:
        cpu: ## e.g. 250m
        memory: ## e.g. 500m
      requests:
        cpu: ## e.g. 75m
        memory: ## e.g. 150m
```

If your Airbyte deployment is underprovisioned, you may notice occasional 'stuck jobs' that remain in-progress for long periods, with eventual failures related to unavailable pods. Increasing job CPU and memory limits may also allow for increased sync speeds.

### Concurrent Sync Limits

To help rightsize Airbyte deployments and reduce the likelihood of stuck syncs, there are configurable limits to the number of syncs that can be run at once:

```yaml
worker:

>   extraEnvs: ## We recommend setting a single value for both environment environment variables.

Reviewer: "We recommend setting a single value for both environment environment variables." This might be less disorienting to read if reworded. Maybe something like "We recommend setting both environments with a single, shared value for the environment variable". This might not be necessary, because the example does help clarify.

>   extraEnvs: ## We recommend setting a single value for both environment environment variables.

Author: ```suggestion
  extraEnvs: ## We recommend setting both environment variables with a single, shared value.
```

    - name: MAX_SYNC_WORKERS
      value: ## e.g. 5
    - name: MAX_CHECK_WORKERS
      value: ## e.g. 5
```

> If you are intending to run many syncs at a time, you may also want to increase the number of worker replicas that run in your Airbyte instance:

Reviewer: ```suggestion
If you intend to run many syncs at the same time, you may also want to increase the number of worker replicas that run in your Airbyte instance:
```
